Fixes typo in the `ORDER BY` clause in a user query.

brettshumaker opened the pull request:
## Description
One of the users queries contained a typo. The `$args['orderby']` was used twice instead of using `$args['order']` for the second arg. This resulted in the `ORDER BY` clause being `ORDER BY 'ID' 'ID'`.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).

brettshumaker committed:
Fixes typo in the `ORDER BY` clause in a user query.

`$args['orderby']` was used twice. The second time should have been `$args['order']`.

WPprodigy reviewed (commented):
Before:

> SELECT SQL_CALC_FOUND_ROWS ID FROM wp_users ORDER BY 'ID' 'ID' LIMIT 0, 350

(vitess not a fan)

After:

> SELECT SQL_CALC_FOUND_ROWS ID FROM wp_users ORDER BY 'ID' 'asc' LIMIT 0, 350

(vitess still not a fan because of quotes around `asc` >.<)

brettshumaker committed:
Sanitize and validate the ORDER BY clause.

Previously, the order by direction was quoted: `ORDER BY 'ID' 'desc'` and Vitess didn't like that.

This passes the `ORDER BY` clause through `sanitize_sql_orderby()`, and if that doesn't return `false`, sets up the clause. We then use that directly in the SQL statement.

WPprodigy reviewed (approved):
Vitess approves now :)

pauarge committed:
Merge branch 'develop' into fix/typo-in-user-query

pauarge committed:
Fixing linting issues

pauarge committed:
Merge branch 'develop' into fix/typo-in-user-query

rinatkhaziev committed:
Merge branch 'develop' into fix/typo-in-user-query
